open-api/rest-catalog-open-api.yaml

```diff
@@ -1756,6 +1871,29 @@ components:
           items:
             $ref: '#/components/schemas/TableUpdate'
 
+    CommitTransactionTableRequest:
+      type: object
+      required:
+        - identifier
+        - requirements
+        - updates
+      properties:
+        identifier:
+          $ref: '#/components/schemas/TableIdentifier'
+        requirements:
+          type: array
+          items:
+            $ref: '#/components/schemas/TableRequirement'
+        updates:
+          type: array
+          items:
+            $ref: '#/components/schemas/TableUpdate'
+
+    CommitTransactionRequest:
+      type: array
```

Comment:
I don't think this should be an array. The problem with using an array is that you can't make top-level additions to the request without changing its type. Using an object allows you to add optional keys later. For example, what happens if we want to create and track a transaction ID? That would need to be added here.

Instead of an array, I think we should use a structure like this:

```yaml
    CommitTransactionRequest:
      type: object
      required:
        - tables
      properties:
        tables:
          type: array
          items:
            description: Table commit request; must provide an `identifier`
            $ref: '#/components/schemas/CommitTableRequest'
```